resources/guides/parking_guide.md

```diff
@@ -0,0 +1,62 @@
+---
+id: parking_guide
+title: Parking Guide
+sidebar_label: Parking Guide
+slug: /guides/parking
+---
+
+_Last updated on 2022/06/10_
+
+This is a guide to the parking spaces around the campus. See this [map](https://maps.mapsted.com/uwindsor) if you would like to see the different UWindsor buildings beforehand.
+
+## Official UWindsor Parking Spaces
+
+There are 17 official parking lots within the UWindsor campus. This includes visitor parking, staff parking, and student parking. You can locate these parking lots in this [campus map](https://web2.uwindsor.ca/pac/campusmap/index.php).
+
+![img](../../static/img/student-parking.png)
```

Comment:
the alt text(the part inside the [..]) should always be a title or description of the image. it is used by screen readers to help people who cannot see the image.

for example if I were to put a photo of a chair, I would write
```md
![photo of chair](../../static/img/chair.jpg)
```
BTW this goes for all images without alt text in this PR(and in general)